build/teamcity-test.sh

```diff
@@ -41,19 +41,24 @@ bundle install
 
 run_cockroach
 
-if ! (bundle exec rake test); then
-    echo "Tests failed"
-    HAS_FAILED=1
-else
-    echo "Tests passed"
-    HAS_FAILED=0
-fi
+source ./build/test_cases.sh
+
+HAS_FAILED=0
+for test_file in ${TEST_CASES[@]}; do
+    if ! (TEST_FILES_AR="$test_file" bundle exec rake test); then
```

Comment:
Hmm actually does this work? Based on the instructions in #76 it seems like for the tests that we had to add/modify, it should be specified with `TEST_FILES` instead?

Reply:
`TEST_FILES_AR` will run the ActiveRecord tests, but you're right--it won't include the ones we've changed. Maybe we can add another loop that goes through `test/cases/**/*test.rb` and sets `TEST_FILES`. Does that make sense?